js/form.js

```diff
@@ -0,0 +1,26 @@
+const form = document.querySelector('[data-js="question-form"]');
+const questionInput = document.querySelector('[data-js="question-input"]');
+const answerInput = document.querySelector('[data-js="answer-input"]');
+const tagsInput = document.querySelector('[data-js="tags-input"]');
+
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+
+  const newQuestionCard = document.createElement("section");
+  newQuestionCard.classList.add("question-card");
+  form.append(newQuestionCard);
+
+  const newQuestion = document.createElement("p");
+  newQuestion.classList.add("question");
+  newQuestion.textContent = questionInput.value;
+  newQuestionCard.append(newQuestion);
+
+  const newAnswer = document.createElement("p");
+  newAnswer.textContent = answerInput.value;
+  newQuestion.append(newAnswer);
+
+  const newTags = document.createElement("li");
+  newTags.classList.add("tags");
+  newTags.textContent = tagsInput.value;
+  newAnswer.append(newTags);
+});
```

Comment:
There is a lot of your question-card code missing in order to create a new question card.
For example you are not creating an ul-Element and currently append the newTags ("li") to newAnswer ("p") which is not valid. 

This is all of your question mark code that you will have to create:
`<section class="question-card">
        <p class="question">What property flips the axes in flexbox?</p>
        <button type="button" class="answer-button" data-js="answer-button">
          Show Answer
        </button>
        <p data-js="answer" class="hidden answer">Answer</p>
        <button type="button" data-js="bookmark-button" class="bookmark-button">
          <svg
            data-js="bookmark-icon"
            xmlns="http://www.w3.org/2000/svg"
            height="48"
            viewBox="0 -960 960 960"
            width="48"
          >
            <path
              d="M200-120v-665q0-24 18-42t42-18h440q24 0 42 18t18 42v665L480-240 200-120Zm60-91 220-93 220 93v-574H260v574Zm0-574h440-440Z"
            />
          </svg>
        </button>
        <ul>
          <li class="tags">#html</li>
          <li class="tags">#css</li>
          <li class="tags">#flexbox</li>
        </ul>
      </section>`

Reply:
It would be nice if you reset your form after submitting the new question. This way the user can fill out the form again adding another question.